src/main/java/appeng/core/features/registries/RecipeHandlerRegistry.java

```diff
@@ -44,7 +44,7 @@
 public class RecipeHandlerRegistry implements IRecipeHandlerRegistry
 {
 	private final Map<String, Class<? extends ICraftHandler>> handlers = new HashMap<>( 20 );
-	private final Collection<ISubItemResolver> resolvers = new LinkedList<>();
+	private final Collection<ISubItemResolver> resolvers = new ArrayList<>();
 
 	@Override
 	public void addNewCraftHandler( final String name, final Class<? extends ICraftHandler> handler )
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "ICraftHandler"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)